scripts/microbe_stage/setup.lua

```diff
@@ -34,6 +34,7 @@ local function setupCamera()
     viewportEntity:addComponent(viewportComponent)
 end
 
+-- there must be some more robust way to script agents than having stuff all over the place.
```

Comment:
Yeah, this is one of the things I'll be opening a new issue for. I think we could get this in for 0.2.5